lookout/style/typos/analyzer.py

```diff
@@ -162,11 +161,20 @@ def generate_typos_fixes(self, changes: Sequence[UnicodeChange]) -> Iterator[Typ
                     ]
                     if identifier_candidates:
                         yield TypoFix(
-                            content=file.content,
+                            content=file.content.decode("utf-8", "replace"),
                             path=file.path,
                             identifier=identifier,
                             line_number=new_identifiers[index].start_position.line,
-                            candidates=identifier_candidates)
+                            candidates=identifier_candidates,
+                            identifiers_number=len(identifiers),
+                        )
+
+    @staticmethod
+    def _get_identifiers(uast, lines):
+        return [node for node in extract_changed_nodes(uast, lines)
+                if (bblfsh.role_id("IDENTIFIER") in node.roles and
```

Comment:
Please extract `bblfsh.role_id("IDENTIFIER")` and `IMPORT` to top function level.

Reply:
I add it to module level.